### DISCUSSION TITLE
Question: iOS Universal builds?

### DISCUSSION THREAD

**Discussion by vijaytholpadi** [Shopify/tophat#5] [Q&A] [answered]
I noticed that the documentation mentions about Universal builds. 

Is this applicable to iOS as well?

In my experience it has not been possible to install an `app` from a device `ipa` file even though both the device and the simulator share the same architecture (arm64)

Apple dev forums also [Install .app file on simulator m1 | Apple Developer Forums](https://developer.apple.com/forums/thread/748650?answerId=783165022#783165022) something similar. But thought I'll clarify.

Any info will be very useful. Thanks.

**Comment by lfroms** [comment, marked as answer]
Great question, @vijaytholpadi!  You are correct.  An iOS Simulator build will only work on simulators (virtual devices), and an iOS build will only work on devices (physical devices).

Launching builds shares the same API for both iOS and Android, but in practice, you'll probably never use `universal` for iOS.
